src/main/java/it/gov/pagopa/pu/debtpositions/service/update/DebtPositionCancelInstallmentServiceImpl.java

```diff
@@ -51,6 +52,14 @@ public String cancelInstallment(DebtPositionDTO debtPositionDTO, List<Installmen
 
   @Override
   protected void applyOperation(DebtPositionDTO debtPositionDTO, List<InstallmentDTO> installments2operate, String accessToken, Organization org) {
+    List<String> installmentsNonCancellable = installments2operate.stream()
+      .filter(installmentDTO -> !InstallmentUtils.MODIFIABLE_STATUSES.contains(installmentDTO.getStatus()))
```

Comment:
just CANCELLED installment could be ignored, other statuses will be errors
```suggestion
      .filter(installmentDTO -> !InstallmentStatus.CANCELLED.equals(installmentDTO.getStatus()))
      .map(installmentDTO -> {
         if(!InstallmentUtils.MODIFIABLE_STATUSES.contains(installmentDTO.getStatus())) {
            throw new ConflictErrorException("The installment with id " + installmentDTO.getInstallmentId() + " cannot be cancelled because is not in allowed status: " + installmentDTO.getStatus());
         }
         return installmentDTO;
      })
```